blocksuite/affine/data-view/src/view-presets/chart/pc/chart-view-ui.ts

```
@@ -0,0 +1,585 @@
+import { signal, computed } from '@preact/signals-core';
+import { renderUniLit } from '../../../core/index.js';
+import Chart from 'chart.js/auto';
+import { chartContainerStyle } from '../styles.js';
+import { tableViewStyle } from '../../table/pc/table-view-style.js';
+import { TableSingleView } from '../../table/table-view-manager.js';
+import { tableViewModel } from '../../table/define.js';
+import { TableViewUILogic } from '../../table/pc/table-view-ui-logic.js';
+import type { TableViewData } from '../../table/define.js';
+import { DEFAULT_COLUMN_WIDTH } from '../../table/consts.js';
+import type { FilterGroup } from '../../../core/filter/types.js';
+import type { ViewManager } from '../../../core/view-manager/view-manager.js';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)